models/instance_segmentation/model_templates/coco-instance-segmentation/instance-segmentation-0002/model.py

```diff
@@ -35,7 +35,7 @@
         type='StandardRoIHead',
         bbox_roi_extractor=dict(
             type='SingleRoIExtractor',
-            roi_layer=dict(type='RoIAlign', out_size=7, sample_num=0),
+            roi_layer=dict(type='RoIAlign', output_size=7, sample_num=0),
```

Comment:
Here and below should be `sampling_ratio` instead `sample_num`.

Reply:
Yeah, thanks. I have it in the list, but it's not the main issue so far.